dlio_benchmark/checkpointing/tf_checkpointing.py

```diff
@@ -42,11 +58,11 @@ def __init__(self):
         super().__init__("pb")
 
     @dlp.log
-    def get_tensor(self, size):
-        return tf.random.uniform((int(size / 4),), maxval=100, dtype=tf.dtypes.int32)
+    def get_tensor(self, size, datatype="int8"):
+        return tf.ones((size), dtype=get_tf_datatype(datatype))
```

Comment:
Same comment as PyTorch is to make sure tensor representation has same footprint.